Makefile

```diff
@@ -117,9 +117,44 @@ deploy-local-easy:
 	./build/deploy_local.sh
 
 # Generate manifests e.g. CRD, RBAC etc.
-manifests: controller-gen
+manifests: controller-gen kustomize
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./..." output:crd:artifacts:config=config/crd/bases
 
+	# Create directories for manifests if they do not exist
+	mkdir -p config/deploy/kubernetes
+	mkdir -p config/deploy/openshift
+
+	# Generate kubernetes.yaml
+	helm template dynatrace-operator config/helm/chart/default --namespace dynatrace --set platform="kubernetes" --set autoCreateSecret=false --set operator.image="quay.io/dynatrace/dynatrace-operator:snapshot" > config/deploy/kubernetes/kubernetes.yaml
+	sed -i '/app.kubernetes.io/d' config/deploy/kubernetes/kubernetes.yaml
```

Comment:
does not work on mac
replaced it with:
```
sed -i '.original' -e ...
```
seems to have fixed it

Reply:
related: https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux